### PR TITLE
Config for sandbox execution

### PR DESCRIPTION
**Pull request by ArulselvanMadhavan**
My attempt to work on https://github.com/fthomas/scala-steward/issues/180 

I can get the tests to compile in my machine but the tests fail when they try to read files with test data like "create-fork.json"

Submitting this for initial feedback.

## TODO:
1. I need to figure out how to get the tests to pass in my machine
2. Need to write unit test for this new config flag.
3. I am using scalafmt 1.5.1 . I ran it against both src and test. Still I can't get past the scalafmtCheck in travis CI.

### CODE REVIEW

**Comment by ArulselvanMadhavan** [comment]
retest

File: modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala [hunk 1]
```diff
@@ -93,7 +95,11 @@ object SbtAlg {
         fileAlg.home.map(_ / ".sbt")
 
       def exec(command: Nel[String], repoDir: File): F[List[String]] =
-        ignoreOptsFiles(repoDir)(processAlg.execSandboxed(command, repoDir))
+        if (config.execSandbox) {
+          ignoreOptsFiles(repoDir)(processAlg.execSandboxed(command, repoDir))
+        } else {
+          ignoreOptsFiles(repoDir)(processAlg.exec(command, repoDir))
+        }
```
Comment (fthomas) [inline modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala]: Can we move this conditional to `ProcessAlg`? If `config.execSandbox` is false `ProcessAlg.exexSandboxed` should just be an alias for `ProcessAlg.exec`.

File: modules/core/src/main/scala/org/scalasteward/core/update/UpdateService.scala [hunk 2]
```diff
@@ -108,7 +108,7 @@ class UpdateService[F[_]](
               ignUpdate =>
                 update.groupId == ignUpdate.groupId && update.nextVersion == ignUpdate.nextVersion &&
                   ignUpdate.artifactIds.exists(id => update.artifactId.startsWith(id))
-            )
+          )
         )
```
Comment (fthomas) [inline modules/core/src/main/scala/org/scalasteward/core/update/UpdateService.scala]: If you run `sbt scalafmt` it should reformat the code such that `sbt scalafmtCheck` is happy.

File: modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala (hunk 1 above)
Comment (ArulselvanMadhavan) [reply modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala]: Made the changes. 👍

**Pull request merged**